### ISSUE TITLE
deploy fails because page was not refreshed

### ISSUE THREAD

**Issue by BigMichi1** [bsorrentino/maven-confluence-plugin#225]
sometimes we have a strange issue when we are deploying the pages to confluence which results in the following error during deploy
```[INFO] [ERROR] Failed to execute goal org.bsc.maven:confluence-reporting-maven-plugin:6.9:deploy (deploy) on project checkstyle: error generating report: error: update page
[INFO] [ERROR] Response{protocol=h2, code=501, message=, url=https://confluence.mydomain.de/rest/api/content/361499519}
[INFO] [ERROR] {"statusCode":501,"message":"Unable to save changes to unreconciled page ContentId{id=361499519}. Refreshing the page should fix this.","reason":"Not Implemented"}
[INFO] [ERROR] -> [Help 1]
```

when this happens we found out that simply opening the page with the browser fixes this problem, so it means every time we are getting the error we open the link provided in the message which response with a JSON containing the name of the page we need to open with a web-browser.

it would be really really helpful if this step can be included as part of handling deploy failures, getting the URL from the failure response and doing a get call to this page

**Comment by bsorrentino** [comment]
Hi @BigMichi1 thanks for feedback

I never dealt with this situation ... I'll working on it asap

**Comment by BigMichi1** [comment]
we are hitting this issue https://jira.atlassian.com/browse/CONFSERVER-59196

**Comment by bsorrentino** [comment]
This issue seems fixed in confluence version 7.3.3.

Currently retry policy is not in the roadmap of this plugin so I'll not work on it ... feel free to submit a PR with your suggested fix
